src/viam/app/data_client.py

```diff
@@ -298,6 +300,39 @@ async def tabular_data_by_mql(self, organization_id: str, mql_binary: List[bytes
         response: TabularDataByMQLResponse = await self._data_client.TabularDataByMQL(request, metadata=self._metadata)
         return [bson.decode(bson_bytes) for bson_bytes in response.raw_data]
 
+    async def get_latest_tabular_data(self, part_id: str, resource_name: str, resource_subtype: str, method_name: str) -> Optional[Tuple[Dict[str, ValueTypes], datetime, datetime]]:
```

Comment:
Could you reorder the returned tuple to mirror the api message field ordering? Would be good to be consistent with order across SDKs. ie Tuple[datetime, datetime, Dict[str, ValueTypes]]. Here and below!